docs/detections/rules-ui-manage.asciidoc

```diff
@@ -34,6 +34,10 @@ To filter the rules list, enter a search term in the search bar and press **Retu
 
 NOTE: Searches for index patterns and MITRE ATT&CK tactics and techniques must match exactly, are case sensitive, and do _not_ support wildcards. For example, to find rules using the `filebeat-*` index pattern, the search term `filebeat-*` is valid, but `filebeat` and `file*` are not because they don't exactly match the index pattern. Likewise, the MITRE ATT&CK tactic `Defense Evasion` is valid, but `Defense`, `defense evasion`, and `Defense*` are not.
 
+You can also filter the rules list by selecting the *Tags*, *Elastic rules*, *Custom rules*, *Enabled rules*, and *Disabled rules* filters next to the search bar.
```

Comment:
@jmikell821 do you think the enabled and disabled UI text will be changed in 8.7? Or is this copy pretty much set for the release?